x-pack/plugins/apm/server/routes/suggestions/route.ts

```diff
@@ -30,16 +38,58 @@ const suggestionsRoute = createApmServerRoute({
     const size = await context.core.uiSettings.client.get<number>(
       maxSuggestions
     );
+
     const suggestions = await getSuggestions({
-      field,
+      fieldName,
+      fieldValue,
       searchAggregatedTransactions,
       setup,
       size,
-      string,
+      start,
+      end,
+    });
+
+    return suggestions;
+  },
+});
+
+const suggestionsWithTermsRoute = createApmServerRoute({
+  endpoint: 'GET /internal/apm/suggestions_by_service_name',
+  params: t.type({
+    query: t.intersection([
+      t.type({
+        fieldName: t.string,
+        fieldValue: t.string,
```

Comment:
I'm not particularly convinced by fieldName/fieldValue. IMHO `kuery` would be more useful here. `fieldName` and `fieldValue` implies that we can only filter on one field, `kuery` is more flexible and familiar to us, and considering we don't validate fieldName/value other than being a string, I feel that is the better option.